src/components/MainMap.tsx

```diff
@@ -98,14 +99,38 @@ export const MainMap = ({
         <div ref={geocoderContainerRef} className="z-1 fixed top-2 right-12" />
         <Source type="geojson" data={sidewalkFeatureCollection}>
           <Layer
-            id="Data"
+            id="Paths"
             type="line"
+            filter={[
+              "match",
+                ["get", "highway"],
+                paths,
+                true,
+                false
+              ]} // Filter for true paths
```

Comment:
What do the positional arguments of true / false do here? I found documentation for filter & match but I wasn't able to figure out what those do

Reply:
It took me a while to figure out the filters. It looks like it's possible to specify the return value independently for each match. In our case, I'm returning true for ANY match in the paths array. It could be more appropriately formatted like this. https://docs.mapbox.com/style-spec/reference/expressions/#match

```
filter={[
              "match",
                ["get", "highway"],
                paths, true, // return true if match found in the array of recognized path values
                false // return false as a fallback
              ]} // Filter for true paths
```